--- Makefile.in_old	2015-06-22 11:29:49.000000000 -0500
+++ Makefile.in	2015-06-22 11:30:03.000000000 -0500
@@ -1343,18 +1343,6 @@
 @HAVE_PYTHON_TRUE@maintainer-clean-local:
 @HAVE_PYTHON_TRUE@	rm -rf "$(srcdir)/fused-src"
 
-# Disables 'make install' as installing a compiled version of Google
-# Test can lead to undefined behavior due to violation of the
-# One-Definition Rule.
-
-install-exec-local:
-	echo "'make install' is dangerous and not supported. Instead, see README for how to integrate Google Test into your build system."
-	false
-
-install-data-local:
-	echo "'make install' is dangerous and not supported. Instead, see README for how to integrate Google Test into your build system."
-	false
-
 # Tell versions [3.59,3.63) of GNU make to not export all variables.
 # Otherwise a system limit (for SysV at least) may be exceeded.
 .NOEXPORT:
